src/commands/provider/remove-units.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { color } from "@oclif/color";
+import { Flags } from "@oclif/core";
+
+import { BaseCommand, baseFlags } from "../../baseCommand.js";
+import { commandObj } from "../../lib/commandObj.js";
+import { ENV_FLAG, PRIV_KEY_FLAG } from "../../lib/const.js";
+import { initCli } from "../../lib/lifeCycle.js";
+import { input } from "../../lib/prompt.js";
+import {
+  ensureChainNetwork,
+  getSigner,
+  promptConfirmTx,
+  waitTx,
+} from "../../lib/provider.js";
+
+export default class RemoveUnits extends BaseCommand<typeof RemoveUnits> {
+  static override description =
+    "Sub units to specific nox instance as a Compute Peer";
+  static override flags = {
+    ...baseFlags,
+    ...PRIV_KEY_FLAG,
+    ...ENV_FLAG,
+    "peer-id": Flags.string({
+      description:
+        "Peer id of the nox instance that you want to register as a Compute Peer",
+      helpValue: "<peer-id>",
+    }),
+    units: Flags.string({
+      description: "Number of available worker units on this Compute Peer",
+      helpValue: "<number>",
+    }),
+  };
+
+  async run(): Promise<void> {
+    const { flags, maybeFluenceConfig } = await initCli(
+      this,
+      await this.parse(RemoveUnits),
+    );
+
+    const network = await ensureChainNetwork(flags.env, maybeFluenceConfig);
+
+    const unitsCount =
+      flags.units ?? (await input({ message: "Enter new units count" }));
+
+    const peerId =
+      flags["peer-id"] ?? (await input({ message: "Enter peerId" }));
+
+    const [{ DealClient }, { digest }, { base58btc }] = await Promise.all([
+      import("@fluencelabs/deal-aurora"),
+      import("multiformats"),
+      // eslint-disable-next-line import/extensions
+      import("multiformats/bases/base58"),
```

Comment:
Maybe we can import multiformats statically?
@shamsartem

Reply:
we can import everything statically, but the less static imports - the faster cli works as per my experiments

Reply:
There is still stuff that is imported statically but I tried to remove what I can - and each lib that I removed improved speed a bit